sdk/core/core-http/lib/proxyAgent.ts

```diff
@@ -13,14 +14,16 @@ export type ProxyAgent = { isHttps: boolean; agent: http.Agent | https.Agent };
 export function createProxyAgent(
   requestUrl: string,
   proxySettings: ProxySettings,
-  headers?: HttpHeaders
+  headers?: HttpHeaders,
+  tlsOptions?: tls.TlsOptions
```

Comment:
How do we feel about adding a new optional param here?  Might require a minor version bump in the next release.

Reply:
Turns out this function is internal to the library so versioning won't be an issue!